code/game/objects/items/robot/robot_items.dm

```diff
@@ -120,12 +120,14 @@
 			if(!shock_cooldown)
 				if(ishuman(M)||ismonkey(M))
 					M.electrocute_act(15, user, 1)
+					add_logs(user, M, "eletrocuted with \the [src]", admin = (user.ckey && M.ckey) ? TRUE : FALSE)
```

Comment:
The ternary is redundant.
You have a statement, ``user.ckey && M.ckey`` that returns true/false. 
What you're doing is saying "If the statement is true, return true. otherwise return false" When you could just return the value of the statement itself.